src/role_sourcer.js

```diff
@@ -14,25 +14,24 @@
 roles.sourcer = {};
 
 roles.sourcer.settings = {
-  param: ['controller.level'],
+  param: ['energyCapacityAvailable'],
   prefixString: {
-    1: 'MW',
-    3: 'MWC'
+    150: 'MW',
+    600: 'MWC'
   },
   layoutString: {
-    2: 'W',
-    3: 'MW',
-    5: 'MW'
+    0: 'W',
```

Comment:
Not sure what you mean. from 0 to 649 the `layoutString` should be `'W'`, well I could also put `150`, but `0` makes clear from the beginning.